b2sdk/file_version.py

```diff
@@ -255,3 +257,60 @@ def __init__(self, file_id, file_name):
     def as_dict(self):
         """ represents the object as a dict which looks almost exactly like the raw api output for delete_file_version """
         return {'action': 'delete', 'fileId': self.file_id, 'fileName': self.file_name}
+
+
+class FileIdNameAndRetention(object):
+    """
+    A structure which represents a B2 cloud file with just `fileName`, `fileId` and `fileRetention` attributes.
+
+    Used to return data from calls to :py:meth:`b2sdk.v1.B2Api.update_file_retention`.
```

Comment:
it says back _to when_ the file is going to be retained, something that may be important for the caller and something that caller can not reliably predict before the call. It depends on the date on the server during request processing.